src/routers/AppRouter.js

```diff
@@ -83,12 +84,15 @@ const AppRouter = () => {
           <ErrorPage code={500} />
         </Route>
         <Route path={RoutePath.HOME_PAGE} exact>
+          <MobileMenuBar title="Teethcare" />
```

Comment:
lmao! why do you need to do this ?????? only need to put it at the top and then chẹck the location.pathname from useLocation